Packs/CommonTypes/IndicatorTypes/reputation-Malware.json

```
@@ -0,0 +1,27 @@
+{
+    "contextPath": "",
+    "contextValue": "",
+    "defaultMapping": null,
+    "details": "Malware V2",
+    "disabled": false,
+    "enhancementScriptNames": [],
+    "excludedBrands": [],
+    "expiration": 0,
+    "file": false,
+    "fileHashesPriority": null,
+    "formatScript": "",
+    "id": "Malware V2",
+    "locked": false,
+    "manualMapping": {},
+    "mergeContext": false,
+    "regex": "",
+    "reputationCommand": "",
+    "reputationScriptName": "",
+    "sortValues": null,
+    "system": false,
+    "updateAfter": 0,
+    "version": -1,
+    "fromVersion": "5.5.0",
+    "layout": "Malware"
```

```suggestion
    "layout": "Malware V2"
```
solves the conflict with malware incident type lyout